PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -147,8 +163,26 @@ function Invoke-SCuBA {
             return
         }
 
-        if ($ProductNames -eq '*'){
-            $ProductNames = "teams", "exo", "defender", "aad", "sharepoint", "onedrive", "powerplatform"
+        if ($PSCmdlet.ParameterSetName -eq 'Report'){
+
+            if ($ProductNames -eq '*'){
+                $ProductNames = "teams", "exo", "defender", "aad", "sharepoint", "onedrive", "powerplatform"
+            }
+
+            $ScubaConfig.ProductNames = $ProductNames | Sort-Object
+            $ScubaConfig.M365Environment = $M365Environment
+            $ScubaConfig.OPAPath = $OPAPath
+            $ScubaConfig.LogIn = $LogIn
+            $ScubaConfig.DisconnectOnExit = $DisconnectOnExit
+            $ScubaConfig.OutPath = $OutPath
+            $ScubaConfig.OutFolderName = $OutFolderName
+            $ScubaConfig.OutProviderFileName = $OutProviderFileName
+            $ScubaConfig.OutRegoFileName = $OutRegoFileName
+            $ScubaConfig.OutReportName = $OutReportName
+        }
+
+        if ($PSCmdlet.ParameterSetName -eq 'Import-Configuration'){
+            Get-ScubaConfig -Path $ConfigFilePath
         }
```

Comment:
This was at my request actually.  Crutch originally did it the way you had in mind.  My thinking was that by biting this bullet now, it would set us up for the future to just pass $ScubaConfig around and be able to add things to it without having to plumb new params (I tend to have an alergic reaction to functions with more than about 4 arguments).

No strong feelings though if you all want to go in a different direction.